waspc/src/Wasp/Generator/WebAppGenerator/AuthG.hs

```diff
@@ -54,17 +56,21 @@ genAuthForms auth =
   ]
 
 genLoginForm :: Wasp.Auth.Auth -> FileDraft
-genLoginForm auth = [relfile|auth/forms/Login.js|] `compiledWithData` ["onAuthSucceededRedirectTo" .= (Wasp.Auth._onAuthSucceededRedirectTo auth)]
+genLoginForm auth =
+  [relfile|auth/forms/Login.js|]
+    `compileTmplToSamePath` ["onAuthSucceededRedirectTo" .= (Wasp.Auth._onAuthSucceededRedirectTo auth)]
```

Comment:
Due to changing the name of the function, I think infix notation is not so suitable any more.